include/fastrtps/Domain.h

```diff
@@ -20,7 +20,7 @@
 #ifndef DOMAIN_H_
 #define DOMAIN_H_
 
-#include "attributes/ParticipantAttributes.h"
+#include <fastrtps/attributes/ParticipantAttributes.h>
```

Comment:
Using quotes is faster for preprocessor. Mantain quotes
```suggestion
#include "attributes/ParticipantAttributes.h"
```

Reply:
But that can cause problems when mocking, in this case, if ParticipantAttributes is mocked (I don't remember if it is the case)

Reply:
Remember we found a solution when a class was mocking.